signing/signing.go

```diff
@@ -101,13 +106,15 @@ func ProcessSignature(jsonPayload []byte, repoName, repoRef string) error {
 
 	// Call scorecard-webapp-api to process and upload signature.
 	// Setup HTTP request and context.
-	url := "https://api.securityscorecards.dev/verify"
-	req, err := http.NewRequest("POST", url, bytes.NewBuffer(payloadBytes)) //nolint
+	rawURL := fmt.Sprintf("https://api.securityscorecards.dev/projects/github.com/%s", repoName)
+	//parsedURL, err := url.Parse(rawURL)
+	//if err != nil {
+	//	return fmt.Errorf("error during urrl.Parse: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("parsing scorecard API endpoint: %w", err)
```

or some language like that.
Reasoning: the return value is already an error, so it's redundant to use in the message. `<action>: <error>` is sufficient. Also, general preference is not reference the function call `url.Parse()`, because the function may change in the future (and then the text will be incorrect).

Reply:
Updated, PTAL.